### PR TITLE
ref(spans): Create script for generating fake traces

### PR DESCRIPTION
**Pull request by evanh**
Create a script that generates fake traces for a set of projects.

All the traces for a single organization (1123) that spans 3 projects. Conceptually it is meant to
mimic a frontend, backend and service project.

The script can write to a file, output to stdout or produce to a kafka topic. It's all random, so
there are no guarantees of how many transactions/spans will actually appear in a given trace. There
is an option to run the script until a given number of spans are generated. Note this isn't exact,
the script will run until at least that many spans are generated.

### CODE REVIEW

**Review by untitaker — APPROVED** [review]
idk where this script is best put, but i think it could go eitehr into snuba or the sandbox. in the sandbox you have the advantage of building some adhoc deployment scripts too

File: scripts/send_spans.py [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+            add_datetimes=add_datetimes,
+        )
+
+        to_split_or_not_to_split = random.random()
+        if span_count < 100 and to_split_or_not_to_split < 0.3:
+            subspans = []
+            subduration = 0
+            if project_id != 3 and to_split_or_not_to_split < 0.15:
+                if verbose:
+                    print(f"{prefix}T{project_id}")
```
Comment (volokluev) [inline scripts/send_spans.py:179]: I feel like log level is a better way to do this

**Review by volokluev — APPROVED** [review]
just a few small comments

File: scripts/send_spans.py [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+    ("tag1", partial(random.randint, 0, 10)),
+    ("tag2", lambda: str(uuid.uuid4())),
+    ("tag3", partial(random.randint, 0, 100000)),
+]
+
+Span = dict[str, Any]
+Producer = Callable[[list[Span], bool, bool], None]
+
+
+def custom_tags(project_id: int) -> dict[str, str]:
```
[the remark this answers is not in the document]
Comment (evanh) [reply scripts/send_spans.py:42]: That's probably fair. I'll change it.

File: scripts/send_spans.py (hunk 1 above)
Comment (evanh) [reply scripts/send_spans.py:185]: I agree, I just stopped improving this basically.

**Pull request merged**